Port's Hyperstation's Blast Door sprites

githubuser4141 opened the pull request:
## About The Pull Request
Simply put, ports the blast door sprites from Hyperstation. IMO it's better fitting, our current blast doors resemble something you'd see outdoors or in a silo.

Also, adds a couple missing blast doors to various (Moesci, Moemed, IH & Cargo) lockdowns, ensuring you'll need at least a space suit and/or a wall-breaking tool to bypass it compared to simply walking through.

Changes the IH lobby lockdown and adds a lockdown checkpoint on the Bridge, screenshots below.


![Bridgecheckpoint1](https://user-images.githubusercontent.com/61243846/198856660-a858d7b5-709e-438d-8d32-3026ff1383da.png)

![IHblastdoors1](https://user-images.githubusercontent.com/61243846/198856663-a3be2af2-ca92-44fa-9852-b62694d771b4.png)


## Why It's Good For The Game

Looks better? (???)


## Changelog
:cl:
add: Added new things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

DiaFRAME444 commented:
Screenshot of sprites?

githubuser4141 commented:
Should be in the diff bot.

ValoTheValo commented:
what do they look like closed and in game?

githubuser4141 commented:
> what do they look like closed and in game?

Here:
![ihshutters](https://user-images.githubusercontent.com/61243846/198856888-adb183f6-9d80-41a5-b3aa-366448ded04a.png)

![ihshutters1](https://user-images.githubusercontent.com/61243846/198856906-99d8fc05-b468-4b5c-8c7a-19f685bd5acd.png)

![shuttersclosed](https://user-images.githubusercontent.com/61243846/198856910-7b53581b-0b09-4861-95c8-4a47e6c40aed.png)

Shutters open over windows are a bit uh, too much. I'll probably revert that.

ValoTheValo commented:
yeah dont really like the look of those to be honest

DiaFRAME444 commented:
Hard style clash, those seem like they're made for servers with Mojave Sun's Infra walls.

githubuser4141 commented:
> yeah dont really like the look of those to be honest

I mean, it fits in better then the current things:

![oldshutters1](https://user-images.githubusercontent.com/61243846/198857319-a6e193f8-a9bd-4aa8-a41c-cc82bd73273e.png)

![oldshutters2](https://user-images.githubusercontent.com/61243846/198857322-d0dc062c-7de8-461c-b93d-2e9d955c319c.png)

githubuser4141 commented:
> Hard style clash, those seem like they're made for servers with Mojave Sun's Infra walls.

It's nothing compared to the current blast door sprites :0

ValoTheValo commented:
> > yeah dont really like the look of those to be honest
> 
> I mean, it fits in better then the current things:
> 
> ![oldshutters1](https://user-images.githubusercontent.com/61243846/198857319-a6e193f8-a9bd-4aa8-a41c-cc82bd73273e.png)
> 
> ![oldshutters2](https://user-images.githubusercontent.com/61243846/198857322-d0dc062c-7de8-461c-b93d-2e9d955c319c.png)

not really

DiaFRAME444 commented:
I don't quite agree with what you've said here but I'm going to skip that and get to the important point.
The sprites just look worse, overall. They're less detailed, the "shading" and everything looks simpler in a bad way. The current sprites look imposing in their own right.

githubuser4141 commented:
> I don't quite agree with what you've said here but I'm going to skip that and get to the important point. The sprites just look worse, overall. They're less detailed, the "shading" and everything looks simpler in a bad way. The current sprites look imposing in their own right.

The current sprites don't even fit with the 3/4th perspective, they're flat, have wildly different colors and a different aesthetic. At least the news ones are 3/4ths and the open/close animation is smoother, better

Alliostra commented:
Firstly there is absolutely no reason for this to be made, secondly it will screw up  the hallways and make all these blast doors more ugly thirdly, one and possibly more sprite palette changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

SirRichardFrancis commented:
Firstly there is absolutely no reason for this to be made, secondly it will screw up the hallways and make all these blast doors more ugly thirdly, one and possibly more sprite palette changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

githubuser4141 commented:
> Firstly there is absolutely no reason for this to be made, secondly it will screw up the hallways and make all these blast doors more ugly thirdly, one and possibly more sprite palette changes are direct violation to contributing guidelines, you should try reading them. https://wiki.cev-eris.com/Content_GuidelinesEn

I mean, if you say so. I don't know why a 3/4ths blast door with the similar steel palette is somehow then a 2D blast door with a wildly different palette with a 40k/post-apoc/grimdark palette buuuuut okay.

DiaFRAME444 commented:
> Updated it using Baystation's 3/4ths blast doors instead.

We have no sprite diff bot so uhh
Pics please?

ValoTheValo commented:
the updated sprites when non extended look really bad

SirRichardFrancis commented:
It's here, my dude.

![image](https://user-images.githubusercontent.com/65828539/198906056-293bc766-7fc2-4865-b70f-1be0c41c80b3.png)

![image](https://user-images.githubusercontent.com/65828539/198906068-87b701f8-c09a-4eea-b8bd-178ed0dd9ca0.png)

![image](https://user-images.githubusercontent.com/65828539/198906082-c2a5d6eb-847c-41bc-abcb-52aefa6bb988.png)

ValoTheValo commented:
> It's here, my dude.
> 
> ![image](https://user-images.githubusercontent.com/65828539/198906056-293bc766-7fc2-4865-b70f-1be0c41c80b3.png)
> 
> ![image](https://user-images.githubusercontent.com/65828539/198906068-87b701f8-c09a-4eea-b8bd-178ed0dd9ca0.png)
> 
> ![image](https://user-images.githubusercontent.com/65828539/198906082-c2a5d6eb-847c-41bc-abcb-52aefa6bb988.png)

those still show the hyperstation ones

SirRichardFrancis commented:
Please do not quote pictures :torment:

githubuser4141 commented:
> 

It's also in the the updated OP.. you scrolled right by it..

SirRichardFrancis commented:
Jokes aside, new shutters might be marginally better, at best, but I personally don't like them, something just feels off.
Also, is there only one direction? Side-shutter is still flat.

Shutters from Hyperstation were complete garbage.

githubuser4141 commented:
> 

The side shutters are flat from what I can tell, yes. The layering is a bit off on on-window sprite which is probably what doesn't look right. That, and there's a bit 'left over' when there's no adjacent wall. :pain

githubuser4141 commented:
> Jokes aside, new shutters might be marginally better, at best, but I personally don't like them, something just feels off. Also, is there only one direction? Side-shutter is still flat.
> 
> Shutters from Hyperstation were complete garbage.

I think you're 100% right about the blast door sprites so I reverted them. Bay's shutter sprites weren't bad looked funky on our grille-less clear window sprites, so I changed them to TG's shutters.

Firefox13 commented:
Are you sure you set the dirs right? I don't think the "up" dir should even be used, it looks odd. Right/left are fine, though
![изображение](https://user-images.githubusercontent.com/57810301/199270532-d2620088-9a8a-4106-8a17-4664d16adf79.png)
Blast door never should be alone - they're either on airlocks or glass, otherwise it looks bad. It also fits badly with our tiles.
![изображение](https://user-images.githubusercontent.com/57810301/199270921-7666342d-332c-4753-b2ea-61e5dfd975e5.png)
![изображение](https://user-images.githubusercontent.com/57810301/199270937-a9155473-73b1-4766-8200-86e5f18cd914.png)
Make sure you put a different kind of tile below table and racks and fix that floor APC
![изображение](https://user-images.githubusercontent.com/57810301/199271106-c2cf88e5-c8e8-47fe-abaf-44b48f072f72.png)

Firefox13 commented:
so... which sprites are final and which are not? I see one kind on icondiffs yet another on your images.

Firefox13 commented:
Also, are you sure you set areas right? You've added a new APC that uses already existing area. You need to create a new one.

githubuser4141 commented:
> Also, are you sure you set areas right? You've added a new APC that uses already existing area. You need to create a new one.

Updated!

<details>
  <summary>Bridge & IH Lobby</summary>
  
![ih1](https://user-images.githubusercontent.com/61243846/199906497-bcbd284f-bf3e-48fe-ad21-453d29c03f8e.png)

![chk1](https://user-images.githubusercontent.com/61243846/199906515-53049f9f-cfec-4fb6-b087-056efda31654.png)

  
</details>

Kurgis commented:
![image](https://user-images.githubusercontent.com/35203040/200680691-08e5879f-67ea-4318-a4e4-a2b4981f084a.png)
Hulling needs to be placed above the new area, I'd also suggest removing the maintenance outcropping.

ValoTheValo commented:
the bridge checkpoint doesn't really look that good to be honest. in department security offices were never really used and why is there a toilet there now??

githubuser4141 commented:
> the bridge checkpoint doesn't really look that good to be honest. in department security offices were never really used and adding one that IH cant even get to without breaking several doors is questionable
> 
> and why is there a toilet there now??

It's parked on the main artery to the bridge, so it should see some use, the department checkpoints are mostly in remote rarely visited areas, and even then they're bullet traps with nowhere to take cover or retreat to

It's not necessarily for IH, it's for the FO/Captain to either man it themselves or assign someone else to it. If the FO/Cap wants IH presence on the bridge, then they can give them access to it. IH aren't supposed to be on the bridge without fo/cap/ihc's permission

Firefox13 commented:
No. This is just *bad*. One-tile hallway, unless maint (and even then it's still bad) is no-go. This is also just a bad and too cramped toilet room in general and is also sharing areas.
![изображение](https://user-images.githubusercontent.com/57810301/200819721-695fe417-95c0-4c62-a356-8efa7a39ba59.png)

This maintenance is *completely* separated and is using bridge hallway area and *also* shares a wall with AI core which is also a bad idea. Also, 2x2 wall block, ***completely haram***
![изображение](https://user-images.githubusercontent.com/57810301/200820787-fc4c6933-1ae3-429a-bc51-8bad5cce55e7.png)

Checkpoint is good but it's still sharing area with hallway, which isn't good. Having a specific button that blocks off maintenance entry is also an odd choice and it shouldn't be one. I think this entire checkpoint is redundant and button "bridge entrance lockdown" should be on the bridge itself, which closes shutters on the *first* bridge doors, not those after the stairs.
![изображение](https://user-images.githubusercontent.com/57810301/200821128-4e4524c5-8f9b-42c2-83b7-adf8863edbb0.png)

This :(
This amount of buttons and blast doors is little extreme, especially those that block maints and their names are a bit misleading.
- "Brig foyer doors" button is a nice idea but completely useless because you don't see doors from that point (doors are *upstairs*) while the button itself in sergeant's office) and it doesn't open second set of doors.
- External lobby lockdown shouldn't block the maintenance access. That's, like, the idea of it. It's a weakpoint. Block it off with a barricade.
- Internal lobby lockdown can and probably should be partially merged with "External lobby lockdown", so one button closes reinforced glass leading to the brig itself, **but not the airlocks leading into the lobby**
- Add a new button, lobby lockdown which simply blocks off those two doors outlined in red. Nuff said, panic button.
Those that I didn't mention (blast doors leading to maint those three shutters (They look really fancy though, I give you that) should be removed.
This setup will reduce amount of confusion and both of those buttons should be duplicated in sergeant's office.
![изображение](https://user-images.githubusercontent.com/57810301/200822703-a5664d66-b098-4560-a9ec-a57226359c02.png)
![изображение](https://user-images.githubusercontent.com/57810301/200824867-46ed7307-08f5-471f-b964-de2060297893.png)

Firefox13 commented:
This shutter (research desk) should have a different dir (down)
![изображение](https://user-images.githubusercontent.com/57810301/200825683-f4259fdb-a27f-419e-8379-ac0c3e6e25c9.png)

This too (MEO office)
![изображение](https://user-images.githubusercontent.com/57810301/200825786-c9baac2c-ab8e-4f86-b717-36ff1b11181f.png)

R&D total lockdown:
Psychologist gets twice the shutters lol
![изображение](https://user-images.githubusercontent.com/57810301/200825949-dd812d17-b182-4148-bb2a-d99d276289fe.png)

Server maintenance too
![изображение](https://user-images.githubusercontent.com/57810301/200826039-ff6719b3-5402-4e72-9662-f0e23639aace.png)

One button should control lower doors and windows, other button should control upper hallway windows
![изображение](https://user-images.githubusercontent.com/57810301/200826193-487b0752-0851-4e21-a21c-03aba00be3ec.png)
![изображение](https://user-images.githubusercontent.com/57810301/200827242-e212f52f-a9e0-4374-9aa6-4ce75cbd1717.png)


Total lockdown completely blocks off pod bay (who goes there anyway? I think it should block its windows) and ignores toxins lab completely (there are no blast doors there)
![изображение](https://user-images.githubusercontent.com/57810301/200826372-de4e002b-c289-4a36-965b-7642f572e084.png)

Upper hallway, double hatch
![изображение](https://user-images.githubusercontent.com/57810301/200827332-662dc9d3-9d4c-48dd-bba4-f12b2df81268.png)

Firefox13 commented:
Also scratch my earlier comment about "it's a weakpoint" you probably actually should block off both of those ironhammer maintenance doors *and* a courtroom with blast doors too

Firefox13 commented:
I'm not sure if this been like this before but *why the hell* it is controlled by same button
![изображение](https://user-images.githubusercontent.com/57810301/200827684-4d9674e6-a51e-42a7-81c9-6536ef710091.png)

Pharmacy counter button controls maintenance shutters which shouldn't even be there. There's also underplating under the table, which is haram.
![изображение](https://user-images.githubusercontent.com/57810301/200827743-80da425d-0bfc-4d1f-8acc-01ea946dc80c.png)

double hatch
![изображение](https://user-images.githubusercontent.com/57810301/200827847-aeb85e43-6e76-4eec-a4f5-1d4c39190641.png)

Man you really hate medbay so much so when the total lockdown is pressed body scanner is *outside* of it.

Firefox13 commented:
Missing blast doors for guild total lockdown
![изображение](https://user-images.githubusercontent.com/57810301/200828606-9c770967-c657-4417-b671-2be899e34b81.png)

This tile needs blast doors otherwise you can just jump in with closed blast doors
![изображение](https://user-images.githubusercontent.com/57810301/200828701-5337cd58-aa8b-45bb-9c22-9fd73ea1f79c.png)

I find it funny that desk is excluded from total lockdown
![изображение](https://user-images.githubusercontent.com/57810301/200828869-14d54689-65dc-4d67-ae3c-08e77c33ecc9.png)

Firefox13 commented:
I'm cautious of how much pain this can bring because of people getting stuck, especially given you also buff them at the same time.

githubuser4141 commented:
A whole lot of the issues you mentioned is/was original content (TM) that wasn't modified in this PR, but I'll go through it when I've got the time. 

That said.. I think the maint shutters/customer shutters + lockdown blast doors is intentional (also OC), the idea being that the person manning the desk would close the weak shutters if there's a rowdy customer that won't go away. If that person's armed with a rifle they're screwed. The 'total lockdown' doors are more durable poddoors that the dpt's Head would close if they felt a need to securely close the dpt.

Firefox13 commented:
Definitely not the toilet one, though.

ValoTheValo commented:
> I'm not sure if this been like this before but _why the hell_ it is controlled by same button ![изображение](https://user-images.githubusercontent.com/57810301/200827684-4d9674e6-a51e-42a7-81c9-6536ef710091.png)

the chem shutters are my fault. i readded a maint pathway there that was cut off at some point but wanted to keep the MBO to Chem route so i put those shutters there as a way to go from one place to another. ive changed my mind on that medbay remap remap since i made it and honestly dont like it much anymore

ValoTheValo commented:
generally the chem shutter fuckery is my fault. and some of the issues like the R&D Lounge shutter has always been there

githubuser4141 commented:
removing the double maint hatches (either one of them) would be a fairly substantial nerf to department security

ValoTheValo commented:
![image](https://user-images.githubusercontent.com/39888997/201008922-72bf1848-dfd9-4183-a3de-2b99d9ba8d6f.png)
why is there a shutter here?

githubuser4141 commented:
Accident, I already made it go away on the new push that I haven't pushed yet

Firefox13 commented:
Given lack of activity this PR will be closed next Thursday (before merge starts).
